doc/prompt_task_map.json

```diff
@@ -0,0 +1,746 @@
+            "prompt": [
+				"GPT-3 style",
+				"MNLI crowdsource",
+				"always/sometimes/never",
+				"based on the previous passage",
+				"can we infer",
+				"claim true/false/inconclusive",
+				"consider always/sometimes/never",
+				"does it follow that",
+				"does this imply",
+				"guaranteed true",
+				"guaranteed/possible/impossible",
+				"justified in saying",
+				"must be true",
+				"should assume",
+				"take the following as truth"
+            ]
+        },
+        {
+            "task": "task1386_anli_r2_entailment",
+            "prompt": [
+				"GPT-3 style",
+				"MNLI crowdsource",
+				"always/sometimes/never",
+				"based on the previous passage",
+				"can we infer",
+				"claim true/false/inconclusive",
+				"consider always/sometimes/never",
+				"does it follow that",
+				"does this imply",
+				"guaranteed true",
+				"guaranteed/possible/impossible",
+				"justified in saying",
+				"must be true",
+				"should assume",
+				"take the following as truth"
+            ]
+        },
+        {
+            "task": "task1387_anli_r3_entailment",
+            "prompt": [
+				"GPT-3 style",
+				"MNLI crowdsource",
+				"always/sometimes/never",
+				"based on the previous passage",
+				"can we infer",
+				"claim true/false/inconclusive",
+				"consider always/sometimes/never",
+				"does it follow that",
+				"does this imply",
+				"guaranteed true",
+				"guaranteed/possible/impossible",
+				"justified in saying",
+				"must be true",
+				"should assume",
+				"take the following as truth"
+            ]
+        }
+    ],
+    "super_glue/cb": [
+        {
+            "task": "task1388_cb_entailment",
+            "prompt": [
+				"GPT-3 style",
+				"MNLI crowdsource",
+				"always/sometimes/never",
+				"based on the previous passage",
+				"can we infer",
+				"claim true/false/inconclusive",
+				"consider always/sometimes/never",
+				"does it follow that",
+				"does this imply",
+				"guaranteed true",
+				"guaranteed/possible/impossible",
+				"justified in saying",
+				"must be true",
+				"should assume",
+				"take the following as truth"
+            ]
+        }
+    ],
+    "glue/rte": [
+        {
+            "task": "task1344_glue_entailment_classification",
+            "prompt": [
+				"does the claim… follow the fact…",
+				"entailment explained",
+				"imply",
+				"imply separated",
+				"mean"
+            ]
+        }
+    ],
+    "winogrande/winogrande_xl": [
+        {
+            "task": "task1391_winogrande_easy_answer_generation",
+            "prompt": [
+				"Replace",
+				"does underscore refer to",
+				"fill in the blank",
+				"stand for",
+				"underscore refer to"
+            ]
+        }
+    ],
+    "winogrande/winogrande_xs": [
+    ],
+    "winogrande/winogrande_s": [
+    ],
+    "winogrande/winogrande_m": [
+    ],
+    "winogrande/winogrande_l": [
+    ],
+    "winogrande/winogrande_debiased": [
+    ],
+    "story_cloze/2016": [
```

Comment:
@yeganehkordi Do you know the differences between the 2016 and 2018 configs? Does T0 use the 2016 config?

Reply:
The data fields are the same among the 2016 and 2018 splits. Instances might be different.
Yes, they have used the 2016 [config for T0](https://github.com/bigscience-workshop/t-zero/blob/2604dd383c72e864b73efb3c5bce19892c7a679c/evaluation/template_list.py#L67).